File: src/main/java/guru/xgm/language/angelscript/psi/AngelScriptTokenSets.java
package guru.xgm.language.angelscript.psi;

import com.intellij.psi.tree.TokenSet;

import static guru.xgm.language.angelscript.psi.AngelScriptTypes.*;

public interface AngelScriptTokenSets {
    TokenSet COMMENTS = TokenSet.create(LINE_COMMENT, BLOCK_COMMENT);
}

File: src/main/kotlin/guru/xgm/language/angelscript/psi/AngelScriptTokenSets.kt
package guru.xgm.language.angelscript.psi

import com.intellij.psi.tree.TokenSet
import guru.xgm.language.angelscript.psi.AngelScriptTypes.*

interface AngelScriptTokenSets {
    companion object {
        val COMMENTS: TokenSet = TokenSet.create(LINE_COMMENT, BLOCK_COMMENT)
    }
}
